mmv1/products/kms/EkmConnection.yaml

```diff
@@ -0,0 +1,163 @@
+
+
+  ~> **Note:** Ekm Connections cannot be deleted from Google Cloud Platform.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Creating a Ekm Connection': 'https://cloud.google.com/kms/docs/create-ekm-connection'
+  api: 'https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.ekmConnections'
+id_format: 'projects/{{project}}/locations/{{location}}/ekmConnections/{{name}}'
+import_format: ['projects/{{project}}/locations/{{location}}/ekmConnections/{{name}}']
+skip_delete: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'kms_ekm_connection_basic'
+    primary_resource_id:
+      'example-ekmconnection'
+      # Because we can't really delete the resource, we get an error of
+      # dangling resources with the auto generated tests. The hand-written
+      # tests do a better job at testing that the key was removed from state
+      # and that no keyrings are left behind since they create (and destroy)
+      # a project just for this test, but they require org level IAM roles
+      # for the test service account.
+    skip_test: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  decoder: templates/terraform/decoders/kms.go.erb
+  encoder: templates/terraform/encoders/send_nil_body.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The location for the EkmConnection.
+      A full list of valid locations can be found by running `gcloud kms locations list`.
+    required: true
+    ignore_read: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      The resource name for the EkmConnection.
+    required: true
+  - !ruby/object:Api::Type::Array
+    name: 'serviceResolvers'
+    description: |
+      A list of ServiceResolvers where the EKM can be reached. There should be one ServiceResolver per EKM replica. Currently, only a single ServiceResolver is supported
+    required: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'serviceDirectoryService'
+          description: |
+            Required. The resource name of the Service Directory service pointing to an EKM replica, in the format projects/*/locations/*/namespaces/*/services/*
+          required: true
+        - !ruby/object:Api::Type::String
+          name: 'hostname'
+          description: |
+            Required. The hostname of the EKM replica used at TLS and HTTP layers.
+          required: true
+        - !ruby/object:Api::Type::Array
+          name: 'serverCertificates'
+          description: |
+            Required. A list of leaf server certificates used to authenticate HTTPS connections to the EKM replica. Currently, a maximum of 10 Certificate is supported.
+          required: true
+          item_type: !ruby/object:Api::Type::NestedObject
+          properties:
```

Comment:
this whole chunk needs to be indented, lines 86-134 need to be tabbed one more time!

Reply:
Fixed.